docs/recipes/nuxt.md

```diff
@@ -0,0 +1,142 @@
+# Testing Vue.js components with Nuxt.js
+
+## Dependencies
+
+- [Require extension hooks](https://github.com/jackmellis/require-extension-hooks):
+	- `npm i --save-dev require-extension-hooks require-extension-hooks-vue require-extension-hooks-babel@beta`
+
+- [browser-env](browser-testing.md)
+	- `npm i --save-dev browser-env`
+
+- [babel-plugin-module-resolver](https://github.com/tleunen/babel-plugin-module-resolver#readme)
+	- `npm i --save-dev babel-plugin-module-resolver`
+
+## Setup
+
+First set up a helper to configure the environment to transpile `.vue` files and run in a browser like environment.
+
+**`package.json`:**
+
+```json
+{
+	"ava": {
+		"require": [
+			"./test/helpers/register.js",
+			"./test/helpers/setup.js"
+		]
+	}
+}
+```
+
+```js
+// ./test/helpers/register.js
+
+require('@babel/register')({
+	// These patterns are relative to the project root directory (where the `package.json` file lives):
+	ignore: [
+		'node_modules/*',
+		'test/*'
+	]
+});
+```
+
+```js
+// ./test/helpers/setup.js
+
+// Setup browser environment
+require('browser-env')();
+const hooks = require('require-extension-hooks');
+const Vue = require('vue');
+
+// Setup Vue.js to remove production tip
+Vue.config.productionTip = false;
+
+// https://github.com/nuxt/create-nuxt-app/issues/180
+window.Date = global.Date = Date;
+
+// Setup `.vue` files to be processed by `require-extension-hooks-vue`
+hooks('vue').plugin('vue').push();
+// Setup `.vue` and `.js` files to be processed by `require-extension-hooks-babel`
+hooks(['vue', 'js']).exclude(({filename}) => filename.match(/\/node_modules\//)).plugin('babel').push();
```

Comment:
Why this *and* `@babel/register`?

Reply:
You're right, @babel/register & hooks are redundant.

- [x] Fixed in 73621a1